SocialCareCaseViewerApi.Tests/V1/Gateways/DatabaseGateway.Tests.cs

```diff
@@ -1408,6 +1408,38 @@ public void UpdatePersonCreatesCorrectCaseHistoryNoteByCallingProcessDataGateway
             result["is_imported"].Should().Be(false);
         }
 
+        [Test]
+        public void GetPersonsByListOfIdsReturnsCorrectPersonRecordsBasedOnGivenIds()
```

Comment:
Maybe a test for documenting what happens when no IDs match?

Reply:
Good shout, I've added a test to cover that.